doc/source/train/distributed-pytorch/checkpoints.rst

```diff
@@ -0,0 +1,244 @@
+.. _train-checkpointing:
+
+Saving and loading checkpoints
```

Comment:
nit: Can we use Title Case for now?

I talked to @angelinalg and we do want to move to sentence case, but it makes more sense to be consistent across the rest of the Ray documentation for now.

Reply:
Another thoughts, I feel this title is a bit inaccurate when I try to add contents for lightning.

At the beginning, I though there would be contents discussing Ray Train's specific tools for checkpoint saving and loading. But actually, this section is discussing:
- how to report a checkpoint to ray train 
- how to resume training with the latest reported ckpt

The actual saving and loading logic is handled by torch/lightning itself. Does "report and retrieve your checkpoint" or something similar make sense?

Reply:
Ok, after an offline discussion with @matt, I agree to keep "Saving and Loading Checkpoints" as the title, since we want to hide the implementation details of ckpt syncing from the user, and let them just use `ray.train.*` api to "save" and "load" checkpoints.